sample_apps/simple_app_python/example_app.py

```diff
@@ -187,14 +215,157 @@ def do_main(args) -> bool:
             print(fnl(), 'Primary admisison cert is not valid\n')
             sys.exit(1)
 
-        # (fixme): This will currently fail as we have not started the
-        # server process, yet.
-        result = channel.init_client_ssl(server_app_host, server_app_port,
-                                         cctm)
-        assert result is False
+        result = client_dispatch(appln_data_dir,
+                                 server_app_host, server_app_port)
+        if result is False:
+            print(fnl(), 'client_dispatch() failed\n')
+            sys.exit(1)
 
     return result
 
+###############################################################################
+def write_cert_pvt_key_to_files(cctm, whoami, data_dir, print_all):
+    """
+    Persist certificate(s) and private-keys that have been established as
+    part of certification. Data from cc_trust_manager{} is written to disk,
+    so that, later, Python interfaces requiring certificates can use this
+    data from file(s).
+    """
+    assert whoami in ('client', 'server')
+
+    print('cc_auth_key_initialized_ = ', cctm.cc_auth_key_initialized_)
+    print('primary_admissions_cert_valid_ = ', cctm.primary_admissions_cert_valid_)
+
+    if whoami == 'client':
+        cert_outfile = data_dir + '/' + CLIENT_ROOT_SIGNED_CERT
+        pkey_outfile = data_dir + '/' + CLIENT_ROOT_SIGNED_KEYF
+    else:   # whoami == 'server'
+        cert_outfile = data_dir + '/' + SERVER_ROOT_SIGNED_CERT
+        pkey_outfile = data_dir + '/' + SERVER_ROOT_SIGNED_KEYF
+
+    ca_root_outfile = data_dir + '/' + ROOT_POLICY_CERT
+    dump_cert_to_file(cctm.serialized_primary_admissions_cert_,
+                      cert_outfile, print_all)
+
+    dump_private_key_to_file(cctm, pkey_outfile, print_all)
+
+    dump_cert_to_file(cctm.serialized_policy_cert_, ca_root_outfile, print_all)
+
+###############################################################################
+def server_dispatch(data_dir, server_app_host, server_app_port):
+    """
+    Start a server process listening on a SSL socket waiting for client input.
+    """
+    print(fnl(), 'Start server process ...\n')
+
+    # Server needs to authenticate the client; hence 'Purpose.CLIENT_AUTH'
+    context = ssl.create_default_context(ssl.Purpose.CLIENT_AUTH)
+    context.verify_mode = ssl.CERT_REQUIRED
+
+    cert_outfilename = data_dir + '/' + SERVER_ROOT_SIGNED_CERT
+    key_outfilename  = data_dir + '/' + SERVER_ROOT_SIGNED_KEYF
```

Comment:
The 'assumption' here is that something else has persisted the required files at the application's (server's) data-dir.

(And likewise at the client's data-dir.)

So this portion of the app will pick up these files from known locations in the data-dir.